packages/babel-parser/src/tokenizer/types.js

```diff
@@ -139,21 +139,21 @@ export const types: { [name: string]: TokenType } = {
   tilde: new TokenType("~", { beforeExpr, prefix, startsExpr }),
   pipeline: createBinop("|>", 0),
   nullishCoalescing: createBinop("??", 1),
-  logicalOR: createBinop("||", 1),
-  logicalAND: createBinop("&&", 2),
-  bitwiseOR: createBinop("|", 3),
-  bitwiseXOR: createBinop("^", 4),
-  bitwiseAND: createBinop("&", 5),
-  equality: createBinop("==/!=/===/!==", 6),
-  relational: createBinop("</>/<=/>=", 7),
-  bitShift: createBinop("<</>>/>>>", 8),
-  plusMin: new TokenType("+/-", { beforeExpr, binop: 9, prefix, startsExpr }),
-  modulo: createBinop("%", 10),
-  star: createBinop("*", 10),
-  slash: createBinop("/", 10),
+  logicalOR: createBinop("||", 2),
```

Comment:
priority of other operators incremented by +1 so that they have more priority over the `nullishCoalescing` operator which is as per spec.

Reply:
It seems like this incrementing of priority broke this test case
```
FAIL  packages/babel-generator/test/index.js
  ● generation/typescript › cast as
```

Reply:
Why is priority important? It can be the same. `a ?? b && c` is an error anyway, regardless of which has higher priority.

Reply:
@nicolo-ribaudo earlier `||` and `??` had same priority which was not right. `??` should have lower priority then `||`.

Reply:
See the discussion at https://github.com/babel/babel/pull/10269#discussion_r307572385.